front/app/containers/Admin/projects/project/analysis/Demographics/AuthorsByDomicile.tsx

```diff
@@ -0,0 +1,174 @@
+
+// Custom tooltip is rendered when the user hovers over a bar
+const CustomTooltip = ({ payload }: { payload?: any }) => {
+  const total = payload[0]?.payload?.total;
+  const filtered = payload[0]?.payload.filtered;
+
+  return (
+    <Box
+      bg={colors.white}
+      boxShadow={defaultStyles.boxShadowHoverBig}
+      p="12px"
+      borderRadius="3px"
+      w="120px"
+    >
+      <Title variant="h6" my="4px">
+        {payload[0]?.payload?.name}
+      </Title>
+      <Text m="2px">
+        {filtered === total ? (
+          <>{total}</>
+        ) : (
+          <>
+            {filtered} / {total}
+          </>
+        )}
+      </Text>
+    </Box>
+  );
+};
+
+type Props = {
+  customFieldId: string;
+};
+
+const AuthorsByDomicile = ({ customFieldId }: Props) => {
+  const localize = useLocalize();
+  const { analysisId } = useParams() as { analysisId: string };
+  const filters = useAnalysisFilterParams();
+  const { data: totalAuthorsByDomicile } = useAuthorsByDomicile({
+    analysisId,
+    queryParams: {},
+  });
+  const { data: filteredAuthorsByDomicile } = useAuthorsByDomicile({
+    analysisId,
+    queryParams: filters,
+  });
+  const { data: options } = useUserCustomFieldsOptions(customFieldId);
+
+  const filterKey = `author_custom_${customFieldId}`;
+
+  const chartData =
+    totalAuthorsByDomicile &&
+    Object.entries(totalAuthorsByDomicile?.data.attributes.series.users).map(
+      ([optionId, count]) => {
+        const option = options?.data.find((o) => o.id === optionId);
+
+        let name;
+        let shortName;
+        let filtered;
+        let optionKey;
+
+        if (optionId === '_blank') {
+          name = 'Unknown';
+          shortName = '?';
+          filtered =
+            filteredAuthorsByDomicile?.data.attributes.series.users[optionId];
+          optionKey = null;
+        } else {
+          name = localize(option?.attributes.title_multiloc) || optionId;
+          shortName = name.slice(0, 2);
+          filtered =
+            filteredAuthorsByDomicile?.data.attributes.series.users[optionId];
+          optionKey = option?.attributes.key;
+        }
+
+        return {
+          name,
+          shortName,
+          total: count,
+          filtered:
+            filteredAuthorsByDomicile?.data.attributes.series.users[optionId],
+          notFiltered: count - (filtered || 0),
+          optionKey,
+        };
+      }
+    );
+
+  const handleClick = (data, _index, event) => {
+    let newFilterValue;
+    if (event.shiftKey) {
+      newFilterValue = xor(filters[filterKey] || [], [data.optionKey]);
+    } else {
+      newFilterValue = [data.optionKey];
+    }
+
+    if (newFilterValue.length === 0) {
+      updateSearchParams({ [filterKey]: undefined });
+    } else {
+      updateSearchParams({ [filterKey]: newFilterValue });
+    }
+  };
+
+  if (!chartData) return null;
+
+  // See https://citizenlabco.slack.com/archives/C05EZTFP46N/p1693389147200259
```

Comment:
Maybe explain what this comment is referring to in addition to the slack link?

Reply:
:+1: Will do